lib/default.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
```

Comment:
This is a little outside the scope of your bug, but this file should be deleted.